src/rabbit_stomp_processor.erl

```diff
@@ -110,7 +110,16 @@ flush_and_die(State) ->
     close_connection(State).
 
 initial_state(Configuration, 
-    {SendFun, ReceiveFun, AdapterInfo, StartHeartbeatFun, SSLLoginName, PeerAddr}) ->
+    {SendFun, ReceiveFun, AdapterInfo0 = #amqp_adapter_info{additional_info=Extra},
+     StartHeartbeatFun, SSLLoginName, PeerAddr}) ->
+  %% STOMP connections use exactly one channel. The frame max is not
+  %% applicable and there is no way to know what client is used.
+  AdapterInfo = AdapterInfo0#amqp_adapter_info{additional_info=[
+       {channels, 1},
+       {channel_max, 1},
+       {frame_max, 'N/A'},
```

Comment:
I think `131072` is the default we have in most clients. Should we use that? a string `frame_max` is not expected by any part of the codebase except for the UI, I think.